internal/pkg/service/service.go

```diff
@@ -65,8 +65,22 @@ func (service *ServiceLibvirt) AddLibvirtFlags() {
 func Setup(service Service) {
 	service.AddFlags()
 
+	defVerbose := "2"
+
 	// set new default verbosity, was set to 0 by glog
-	flag.Set("v", "2")
+	flag.Set("v", defVerbose)
+	flag.Set("logtostderr", "true")
 
 	flag.Parse()
+
+	// borrowed from cdi/apiserver 1.9.5
```

Comment:
/cc @ksimon1 @yanirq FYI. I spent a solid 2 hours to understand this issue and I'd like to save you guys some time.